src/graphics/image.rs

```diff
@@ -1,3 +1,6 @@
+use core::fmt;
+use std::error::Error;
```

Comment:
Have you run rustfmt on your code? It should merge these uses.

Reply:
fixed. It actually caused other changes in seperate files. made a separate commit in this pr to address them.